app/controllers/homes_controller.rb

```diff
@@ -3,7 +3,8 @@ def show
     if signed_out?
       render "pages/landing"
     else
-      @recommended_items = RecommendedItem.all
+      following_and_self = current_user.following + [current_user]
+      @recommended_items = RecommendedItem.where(user: following_and_self)
```

Comment:
What do you think about moving this into `User` or maybe a `UserRecommendations` utility class?